specification/elasticsan/resource-manager/Microsoft.ElasticSan/stable/2023-01-01/elasticsan.json

```diff
@@ -1512,7 +1832,7 @@
         "systemData": {
```

Comment:
>systemData [](http://example.com/codeflow?start=9&length=10)

You won't need to declare this systemData, if you inherit TrackedResource from common-types/v3 or newer, which includes it